src/Chainweb/Chainweb.hs

```diff
@@ -388,7 +388,7 @@ withChainwebInternal conf logger peer serviceSock rocksDb pactDbDir backupDir re
             let mcfg = validatingMempoolConfig cid v (_configBlockGasLimit conf) (_configMinGasPrice conf)
             -- NOTE: the gas limit may be set based on block height in future, so this approach may not be valid.
             let maxGasLimit = fromIntegral <$> maxBlockGasLimit v maxBound
-            when (Just (_configBlockGasLimit conf) > maxGasLimit) $
+            unless (maybe True (\limit -> _configBlockGasLimit conf <= limit) maxGasLimit) $
```

Comment:
I get confused reading `unless`, and have a semi-strong preference for `when`. Would it be okay to change this back to `when`, and flip `<=` to `>`?

Reply:
I sympathize, I think this line reads kinda like crap anyway and can be nicer.

Reply:
FWIW I usually prefer:
```
    forM_ maxGasLimit $ \limit ->
      unless (configBlockGasLimit conf <= limit) $
```